Assignment 2.Rmd

```diff
@@ -87,11 +92,54 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
-
+```{r}
+student_sum<- EDGE %>% group_by(to) %>% summarise(sum(count))
+names(student_sum)<-c("id","count")
+teacher_sum<-EDGE %>%group_by(from)%>%summarise(sum(count))
+names(teacher_sum)<-c("id","count")
+vertex_new<-bind_rows(student_sum,teacher_sum)
+vertex_new
+
+vertex1
+g1 <- graph.data.frame(EDGE, directed=TRUE, vertices=vertex_new)
+plot(g1,layout=layout.fruchterman.reingold, 
+     edge.width=EDGE$count,
+     vertex.size = vertex_new$count*1.5)
+```
+```{r}
+plot(g,layout=layout.fruchterman.reingold, vertex.size=EDGE$count*3,
+     vertex.color=VERTEX$gender,
+     edge.width=EDGE$count)
+```
```

Comment:
Great graphs! You may want to change the arrow head size to make it a little more balanced. You can also decrease the size discrepency between the teacher and student vertices by using `log(EDGE$count)` rather than `x 3`.